### PR TITLE
add new report Food Related Report

### PR DESCRIPTION
**Pull request by Gigaszi**
### Description
Add new food related layers and a report containing them. Used for the EXPANSE project analyses.

### Corresponding issue
None

### New or changed dependencies
- None

### Checklist
- [x] I have updated my branch to `main` (e.g. through `git rebase main`)
- [x] My code follows the [style guide](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CONTRIBUTING.md#style-guide) and was checked with [pre-commit](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CONTRIBUTING.md#pre-commit) before committing
- [x] I have commented my code
- [x] I have added sufficient unit and integration [tests](https://github.com/GIScience/ohsome-quality-analyst/blob/main/docs/development_setup.md#tests)
- [x] I have updated the [CHANGELOG.md](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CHANGELOG.md)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+##Current Main
+
+###New Features
```
Comment (matthiasschaub) [inline CHANGELOG.md:5]:
```suggestion
## Current Main

### New Features
```

File: website/website/index.html [hunk 2]
```diff
@@ -108,6 +108,7 @@ <h3>2. Choose data quality report.</h3>
 						<option value="RoadReport">Road Report</option>
 						<option value="MultilevelMappingSaturation">Multilevel Mapping Saturation</option>
 						<option value="MultilevelCurrentness">Multilevel Currentness</option>
+						<option value="FoodRelatedReport">Food Related Report</option>
```
Comment (matthiasschaub) [inline website/website/index.html:111]: Do we want to have this report on the website?

File: workers/ohsome_quality_analyst/ohsome/layer_definitions.yaml [hunk 3]
```diff
@@ -331,3 +331,51 @@ minimal:
   description: A minimal layer definition for testing purposes
   endpoint: elements/count
   filter: building=* and geometry:polygon
+
+local_food_shops:
+  name: Local food shops
+  description: Count of local food shops
+  endpoint: elements/count
+  filter: shop=bakery or shop=butcher or shop=greengrocer or shop=seafood or shop=cheese or shop=dairy
+
```
[the remark this answers is not in the document]
Comment (hn437) [reply workers/ohsome_quality_analyst/ohsome/layer_definitions.yaml:340]: afaik nothin regarding layer definitions has been released as they came up with the definitions for us in november, so I would name the project

**Review by hn437 — COMMENTED** [review]
Removed Report from Website, otherwise approved

**Review by hn437 — APPROVED** [review]

File: website/website/index.html (hunk 2 above)
Comment (hn437) [reply website/website/index.html]: no, I removed it

**Pull request merged**